app/javascript/components/recipes/edit/recipe-edit.vue

```diff
@@ -78,6 +68,45 @@
         <div class="relative w-2/5 xl:w-1/4 py-2">
           <div class="text-gray-600 text-sm absolute bg-gray-50 px-1 left-2 -top-1">
             {{ $t('msg.recipes.preparation') }}
+=======
+      <div class="flex flex-col">
+        <div class="flex items-start justify-between h-16 flex-none self-stretch flex-grow-0 mb-8">
+          <div class="relative w-2/5">
+            <div class="text-gray-600 text-sm absolute bg-gray-50 px-1 left-2 -top-2">
+              {{ $t('msg.recipes.name') }}
+            </div>
```

Comment:
⚠️ **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Expected indentation of 18 spaces but found 12 spaces  vue/html-indent